src/NPacMan.Game/Ghost.cs

```diff
@@ -36,18 +36,11 @@ private Ghost(string name, CellLocation homeLocation, CellLocation currentLocati
 
         public Ghost Move(Game game)
         {
-            var newDirection = CurrentStrategy.GetNextDirection(this, game);
-
-            var newLocation = newDirection switch
-            {
-                Direction.Up => Location.Above,
-                Direction.Down => Location.Below,
-                Direction.Left => Location.Left,
-                Direction.Right => Location.Right,
-                _ => Location
-            };
-
-            return WithNewLocationAndDirection(newLocation, newDirection ?? Direction);
+            var nextDirection = CurrentStrategy.GetNextDirection(this, game);
```

Comment:
Actually....the only time GetNextDirection can return Null is from our test implementations of strategies.

Reply:
That's totally fine 👍